lib/apiservers/portlayer/restapi/handlers/containers_handlers.go

```diff
@@ -40,7 +40,10 @@ import (
 	"github.com/vmware/vic/pkg/version"
 )
 
-const containerWaitTimeout = 3 * time.Minute
+const (
+	containerWaitTimeout    = 3 * time.Minute
+	supportVersionForRename = 3
```

Comment:
You're going to have to update this to 4 since I stole 3 already (its merged) :-)